molecule/disable_inactive_users/tests/test_disable_inactive_users.py

```diff
@@ -0,0 +1,42 @@
+"""Module containing the tests for the disable_trace scenario."""
```

Comment:
Reference the correct scenario name:
```suggestion
"""Module containing the tests for the disable_inactive_users scenario."""
```